packages/service-proxy/test/unit/mixin/service.mixin.unit.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/service-proxy
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, Component, Provider} from '@loopback/core';
+import {expect} from '@loopback/testlab';
+import {Class, ServiceMixin} from '../../../';
+
+// tslint:disable:no-any
+
+describe('RepositoryMixin', () => {
```

Comment:
It should be `ServiceProviderMixin`.

Reply:
See https://github.com/strongloop/loopback-next/pull/1649#discussion_r213006676, I prefer to use `ServiceMixin` instead of `ServiceProviderMixin`. 

In my mind, the mixin is adding APIs for working with services. The fact that services are registered via Providers is just an implementation detail to me.